README.md

```diff
@@ -57,6 +57,18 @@ where
  * `cleanup_interval` (default: 1 hour) is the number of seconds between database cleanups 
  * `metrics_lifetime` (default: 24 hours) is the number of seconds that must pass from creation before each metric can be deleted during cleanup
 
+## Metrics visualisation with Grafana
+
+Metrics stored in the database using `membrane_rtc_engine_timescaledb` can be simply visualized using Grafana.
+To start the dashboard with RTC Engine metrics, mount configuration files, that are in `grafna/provisioning` to the Grafana docker container.
+Eg. if you have cloned this repository in the `/root` directory, add `-v /root/membrane_rtc_engine_timescaledb/grafana/provisioning:/etc/grafana/provisioning` option to your `$ docker run`. Beyond that, you have also set four environment variables for your docker container
```

Comment:
I'd avoid using the exact number of variables to set in case that changes in future:

```suggestion
Eg. if you have cloned this repository in the `/root` directory, add `-v /root/membrane_rtc_engine_timescaledb/grafana/provisioning:/etc/grafana/provisioning` option to your `$ docker run`. Beyond that, you have to set the following environment variables for your docker container:
```